### ISSUE TITLE
Wrong files

### ISSUE THREAD

**Issue by Aleksbbgk** [VoronDesign/Voron-Stealthburner#140]
When you download files from this link: https://github.com/VoronDesign/Voron-Stealthburner/releases/tag/1.0 , motor plate and main body of cw2 are not compatible.
![main and plate](https://github.com/user-attachments/assets/b39c705e-0d5c-4104-9a26-b037cb5aaeec)
![main body compare](https://github.com/user-attachments/assets/e4d58ed5-294e-47f1-add8-10f6d11e028f)

i hope pictures will help.

**Comment by nursereese** [comment]
Yep, I came here after printing the parts and finding the main body pieces incompatible. 
The current release #98 includes the incorrect part. 
I only found the "#101 sb shroud got lost" through a Google search for the correct file.

**Comment by aceofjokers** [comment]
Considering its taken so long for someone to fix this....here are the files if anyone comes looking.
[CW2 body and plate.zip](https://github.com/user-attachments/files/16580032/CW2.body.and.plate.zip)
